packages/docbook_xml42.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Docbook_xml42 < Package
+  description 'Version 4.2 - document type definitions for verification of XML data files against the DocBook rule set'
+  compatibility 'all'
+  homepage 'http://www.docbook.org'
+  version '4.2'
+  source_url 'http://www.oasis-open.org/docbook/xml/4.2/docbook-xml-4.2.zip'
+  source_sha256 'acc4601e4f97a196076b7e64b368d9248b07c7abf26b34a02cca40eeebe60fa2'
+
+  depends_on 'docbook'
+  depends_on 'xmlcatmgr'
+  depends_on 'docbook_xml'
+  depends_on 'docbook_xsl' # Requires the catalog.xml created within this package
+  
+  def self.preinstall
+    system "sed -i -e 's,<!-- .* -->,,g' #{CREW_PREFIX}/etc/xml/catalog.xml"
```

Comment:
This line should go in `def self.prebuild` unless it needs to run for every install.

Reply:
Older version of `/usr/local/etc/xml/catalog.xml` have an HTML comment within them, if this line is present, it causes `xmlcatmgr` to fail.